### PR TITLE
Adds missing stacking machine and laser tag closets to Delta

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Adds missing stacking machine and console to Delta waste room.
Adds missing blue and red laser tag closets to Delta.
## Why It's Good For The Game

Every station should have these.
## Images of changes

laser_tag_closet
stacking_machine

## Testing

- Compiled and found the included machinery/closets.
- Used the stacking machine, all good after some tweaking.
## Changelog
:cl:
fix: Delta station: Stacking machine and console have been added
fix: Delta station: Laser tag closets have been added
/:cl:

### CODE REVIEW

**Review by S34NW — APPROVED** [review]

**Comment by S34NW** [comment]
Good to go once merge conflict addressed

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**
